### ISSUE TITLE
the file ts8/rafts.yaml has arbitrary (and incorrect) detector numbers

### ISSUE THREAD

**Issue by aaronroodman** [lsst/obs_lsst#561]
Posting this in case the policy/ts8/rafts.yaml file is used or needed in the future.  Looking at some TS8 single raft test images uncovered that the detector values in the rafts.yaml file are arbitrary, and do not correspond to the actual detector numbers of the rafts as placed in LSSTCam. 

The attached file has the as-built detector numbers for all rafts, and assigns id0=18 for the two rafts not in the LSSTCam focal plane, RTM-031 in ComCam and RTM-004 the spare raft currently in the SLAC TS8.

[rafts.yaml.txt](https://github.com/user-attachments/files/17313485/rafts.yaml.txt)

**Comment by timj** [comment]
The original TS8 file was made long ago before anything was assembled. I'm happy to change it but @jchiang87 should comment because it would mean that you likely need to rebuild any TS8 repos or at least understand that a new repo should be constructed with the new detector because trying to add some TS8 data to an existing repo is going to be extremely confusing.

**Comment by aaronroodman** [comment]
Yes, understand that. @jchiang87 and I were looking into this.  The only TS8 data in a repo is quite new and uses only the spare raft, RTM-004, whose value I kept the same.  Right now I don't see a need to add any of the old TS8 data to a repo, but if I did we would need this file updated.

**Comment by jchiang87** [comment]
I did a quick look at `/repo/main` and there do appear to be TS8 data there as well.  I didn't follow up on specific RTMs though, but I can post a list here later today.

**Comment by jchiang87** [comment]
The TS8 data in `/repo/main` are just copies of some of the runs in `/repo/ir2`.

**Comment by timj** [comment]
If you want this change to happen please make a Jira ticket.
